### ISSUE TITLE
sourceforge.net

### ISSUE THREAD

**Issue by FadeMind** [AdguardTeam/AdguardFilters#48436]
**Site Sample** | 
----------------|
https://sourceforge.net/projects/ext2fsd/files/Ext2fsd/0.69/

Information                            | Value
---                                    | ---
Operating system:                      | Windows 10 1909 x64 (lastest)
Browser:                               | Chrome x64 (lastest)
AdGuard version:                       | **7.3 NIGHTLY** 
Protection Mode:						| **WFP Driver**
Stealth Mode:					       | disabled

| **=== FILTERS ENABLED ===**
|---|
| **SECTION :: Other**
| CoinBlockerLists by ZeroDot1
| **SECTION :: Ads**
| Adguard Base Filter 
| Peter Lowe's List
| **SECTION :: Privacy**
| Adguard Spy Filter
| **SECTION :: Social**
| Adguard Social Filter
| **SECTION :: Annoyances**
| Adguard Annoyances
| I don't care about cookies
| Fanboy's Cookiemonster
| **SECTION :: Security**
| Malware Domains
| Spam404
| **SECTION :: Customs**
| AlleBlock
| KAD - Przekręty
| Official Polish filters for AdBlock, uBlock Origin & AdGuard
| Polish Anti Adblock Filters
| Polskie Filtry Elementów Irytujących
| Polskie Filtry Społecznościowe

### Issue:

GDPR

<details><summary><b>Screenshot:</b></summary>

![20200123-1579758124-001](https://user-images.githubusercontent.com/9846948/72959757-beeb1200-3dab-11ea-9a10-64db3187d94f.png)

</details><br/>

@AdamWr @Alex-302

**Comment by AdamWr** [comment]
It has been fixed few days ago - https://github.com/AdguardTeam/AdguardFilters/issues/47338 and I can't reproduce it (this popup is closed after one second on my end).

**Comment by BlazDT** [comment]
Same

**Comment by Morku** [comment]
As I mentioned in #47338 and after few days, I have to say: Sometimes it works, sometimes it doesn't. Don't know why. A reload of the page helps here which is not the best solution. (yes, I waited more like 5 seconds, after the page loaded completely).
Good to know it is not just me. I started to think "it has to be my PC then".

**Comment by BlazDT** [comment]
Increased timeout in filter, I guess this should help.

**Comment by krystian3w** [comment]
```
[{
	"Host raw": "http://.sourceforge.net/",
	"Name raw": "euconsent",
	"Path raw": "/",
	"Content raw": "false",
	"Expires": "07-02-2020 00:46:23",
	"Expires raw": "1581032783",
	"Send for": "Any type of connection",
	"Send for raw": "false",
	"HTTP only raw": "false",
	"SameSite raw": "no_restriction",
	"This domain only": "Valid for subdomains",
	"This domain only raw": "false",
	"Store raw": "firefox-default",
	"First Party Domain": ""
},
{
	"Host raw": "http://.sourceforge.net/",
	"Name raw": "eupubconsent",
	"Path raw": "/",
	"Content raw": "false",
	"Expires": "07-02-2020 00:46:23",
	"Expires raw": "1581032783",
	"Send for": "Any type of connection",
	"Send for raw": "false",
	"HTTP only raw": "false",
	"SameSite raw": "no_restriction",
	"This domain only": "Valid for subdomains",
	"This domain only raw": "false",
	"Store raw": "firefox-default",
	"First Party Domain": ""
}]
```

**Comment by Morku** [comment]
@BlazDT 
It's awkward. When I see, the filter got updated, I tried again and did not work.

So I wanted to generate a filter log. Closed browser and always tried again and guess what. I am unable to reproduce. It always works. Maybe increase helped........ maybe not...... maybe it's unrelated. From trying about 20x, it failed one time.
Anyway, personally the waiting time feels to long now.

**BUT**

where I get a 100% failure rate is with the Adguard Extension.
I disabled Windows version, disabled integration mode of the extension and it never works here. Or is the Extension limited here?
I am unable to export the protocoll, so I just copypasted it.
[Adguard Extension Log Sourceforge.txt](https://github.com/AdguardTeam/AdguardFilters/files/4109533/Adguard.Extension.Log.Sourceforge.txt)
